autollm/auto/query_engine.py

```diff
@@ -28,7 +28,7 @@ def create_query_engine(
         system_prompt (str): The system prompt to use for the query engine.
         query_wrapper_prompt (str): The query wrapper prompt to use for the query engine.
         enable_cost_calculator (bool): Flag to enable cost calculator logging.
-        embed_model (BaseEmbedding): The embedding model to use for the query engine. Defaults to OpenAIEmbedding.
+        embed_model (Union[str, EmbedType]): The embedding model to use for generating embeddings. Defaults to OpenAI.
```

Comment:
we should add example usage for custom hf models as `local:intfloat/multilingual-e5-large`